### ISSUE TITLE
When pathing to a waypoint, it only paths to the first in the list

### ISSUE THREAD

**Issue by unsightedmetal6** [cabaletta/baritone#1002]
## Some information
Operating system: Windows 10
Java version: 
Minecraft version: 1.12.2
Baritone version: 4.7
Forge mods (if used): 
Optifine, if that counts

## Exception, error or logs
No error, no exceptions

## How to reproduce
Add your steps to reproduce the issue/bug experienced here.
Create two waypointsby doing "#save" Do "#goto secondWaypoint" It will path to the first waypoint instead.

I was playing on an anarchy server. I have two waypoints: farm and home. It will always path to "farm" even if I type "#goto home" I found this out by doing "#user list" and I saw that waypoint farm was above. If I enable #chatDebug, it will say it is pathing to home when in reality it is pathing to farm.

## Final checklist
- [X] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [X] I have included logs, exceptions and / or steps to reproduce the issue.
- [X] I have not used any OwO's or UwU's in this issue.

**Comment by unsightedmetal6** [comment]
Alright I tried deleting all of my waypoints and making a new one, but when I goto it, it still tries to path to some other coordinate.

So, it does NOT do the top waypoint in the list.

**Comment by Xephorix** [comment]
are the two waypoints both set under the same label? (dumb question)

**Comment by unsightedmetal6** [comment]
> are the two waypoints both set under the same label? (dumb question)

No, they were different: one was "farm" and the other was "home."

But I think it is pathing to somewhere else that isn't even a waypoint. Maybe I had it as a waypoint in the past? I've tried deleting all my waypoints but it still paths to the same place.
